log/sentry.go

```diff
@@ -71,12 +74,32 @@ func InitSentry(cfg SentryConfig) (io.Closer, error) {
 	if cfg.SampleRate != nil && *cfg.SampleRate >= 0 && *cfg.SampleRate <= 1 {
 		sampleRate = *cfg.SampleRate
 	}
+
+	// ensure first stack trace frame that is part of this package will be
+	// marked as not in-app to improve the error display in Sentry
+	beforeSend := func(event *sentry.Event, hint *sentry.EventHint) *sentry.Event {
+		for i, exception := range event.Exception {
+			for j := len(exception.Stacktrace.Frames) - 1; j > 0; j-- {
+				frame := exception.Stacktrace.Frames[j]
+				if frame.Function == "ErrorWithSentry" && frame.Module == "github.com/reddit/baseplate.go/log" {
+					event.Exception[i].Stacktrace.Frames[j].InApp = false
+					break
```

Comment:
and if we only check for the module, we could have multiple frames that's not in-app, so we can't break on the first one either.

Reply:
True, it should be safe to mark them all as not in-app.